packages/react/src/components/PageTitleBar/_page-title-bar.scss

```diff
@@ -1,12 +1,31 @@
+/* stylelint-disable declaration-property-unit-blacklist */
 @import '~carbon-components/scss/globals/scss/vars';
 @import '~carbon-components/scss/globals/scss/typography';
 
 .page-title-bar {
   &-title {
-    display: flex;
-    align-items: center;
     justify-content: space-between;
-
+    &-dynamic {
+      position: var(--bar-title-position);
+      margin: -$spacing-05 0 $spacing-06 $spacing-07;
+    }
+    &--condensed {
+      &-static {
+        margin-left: $spacing-03;
+      }
+      &-before {
+        margin-top: $spacing-07;
+        margin-left: $spacing-03;
+        transition: all 240ms;
```

Comment:
is this custom transition or is this something we can use the Carbon motion mixin? https://www.carbondesignsystem.com/guidelines/motion/overview/

Reply:
Examples of its use [here](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/06202605e433edd3a3a936c783cd895ded205289/packages/react/src/components/ProgressIndicator/_progress-indicator.scss#L88)

Reply:
This is a custom transition, but I have added the duration tokens. Thanks!